src/NuGet.Services.Staging.Search/DatabaseSearchService.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.WebUtilities;
+using Newtonsoft.Json.Linq;
+using NuGet.Services.Staging.Database.Models;
+using NuGet.Services.V3Repository;
+
+namespace NuGet.Services.Staging.Search
+{
+    public class DatabaseSearchService : ISearchService
+    {
+        private readonly StageContext _context;
+        private readonly DatabaseSearchResultsFormatter _formatter;
+        private readonly string _stageId;
+
+        public DatabaseSearchService(StageContext stageContext, V3PathCalculator pathCalculator, string stageId)
+        {
+            if (stageContext == null)
+            {
+                throw new ArgumentNullException(nameof(stageContext));
+            }
+
+            if (pathCalculator == null)
+            {
+                throw new ArgumentNullException(nameof(pathCalculator));
+            }
+
+            _context = stageContext;
+            _formatter = new DatabaseSearchResultsFormatter(pathCalculator);
+            _stageId = stageId;
+        }
+
+        public JObject Search(string query)

```

Comment:
Seems to be the default. Just make sure the json response looks like a oneliner instead of a nicely formatted result :-)